tensorflow/java/src/test/java/org/tensorflow/OperationBuilderTest.java

```diff
@@ -175,6 +188,27 @@ public void addControlInput() {
     }
   }
 
+  private static void testSetAttrShapeList(Shape[] shapes) {
```

Comment:
Somewhat unfortunate that the tests require the use of operations that we are discouraging :). The Dataset API (`tf.data` module in Python) better serves most uses of queues.

Reply:
Yeah I tried to use datasets at first (though I didn't know queues were deprecated) but didn't find an easy way for testing it, if you have an idea, I can change it later.

Reply:
Not deprecated, just that there is less of a reason to use them given that the most common use case is better served by Datasets. Anyway, this is fine for now.